app/models/tag.rb

```diff
@@ -0,0 +1,4 @@
+class Tag < ApplicationRecord
+	has_many :taggings
+	has_many :projects, through: :taggings
```

Comment:
indentation

Reply:
@tachyons I probably missed it cause the file was autogenerated. I have fixed it now.